### ISSUE TITLE
Can't manipulate windows in the node

### ISSUE THREAD

**Issue by ilya747** [ITISFoundation/osparc-issues#254]
## Long story short

When I open a node at 12.5" laptop screen, I can't change sizes of the sub-windows (service tree, outputs, etc) or make them to scale of my screen. For example, in jupyter-neuron notebook, I don't see Delete button in the service tree and I can only reduce the size of this window by pulling the partition, but not increase it. Sizes of other windows cannot be changed as well.
## Expected behaviour

I would like to freely change the size and appearance of all the windows available (like in Matlab). Also it would be very convenient if user has a option to drag the sub-window to the position she/he wants. Also, it would be great if the workspace will automatically scale to the screen size (or having a button to choose).
## Actual behaviour

It's not easy to manipulate sub-sections as they are frozen to screen, and not all the buttons are visible as a result - see the screenshot below. The white area with files to choose is not in place as well.
![image](https://user-images.githubusercontent.com/17727649/84485186-ae243580-ac93-11ea-9cdc-f0565b236241.png)
## Steps to reproduce

I had this issue in Jupyter-neuron notebook. But is looks that it is the same in all the nodes.

<!--
## Your environment
```json
[
  {
    "name": "osparc-simcore",
    "version": "5ed8a8f",
    "url": "https://github.com/ITISFoundation/osparc-simcore/tree/5ed8a8f/"
  },
  {
    "name": "osparc-simcore UI [clean]",
    "version": "a75a741",
    "url": "https://github.com/ITISFoundation/osparc-simcore/tree/a75a741/services/web/client/"
  },
  {
    "name": "qooxdoo-compiler",
    "version": "1.0.0-beta.20200111-0015",
    "url": "https://github.com/qooxdoo/qooxdoo-compiler"
  },
  {
    "name": "osparc",
    "version": "1.0.0",
    "url": ""
  },
  {
    "name": "qooxdoo framework",
    "version": "6.0.0-beta-20200110-0942",
    "url": "http://qooxdoo.org"
  },
  {
    "name": "qx-iconfont-fontawesome5",
    "version": "0.2.0",
    "url": "https://github.com/ITISFoundation/qx-iconfont-fontawesome5"
  },
  {
    "name": "qx-iconfont-material",
    "version": "0.1.7",
    "url": "https://github.com/ITISFoundation/qx-osparc-theme"
  },
  {
    "name": "qx-osparc-theme",
    "version": "0.5.4",
    "url": "https://github.com/ITISFoundation/qx-osparc-theme"
  },
  {
    "name": "socket.io",
    "version": "2.2.0",
    "url": "https://github.com/socketio/socket.io"
  },
  {
    "name": "jsondiffpatch",
    "version": "0.3.11",
    "url": "https://github.com/benjamine/jsondiffpatch"
  },
  {
    "name": "jsonTreeViewer",
    "version": "0.6.0",
    "url": "https://github.com/summerstyle/jsonTreeViewer"
  },
  {
    "name": "DOMPurify",
    "version": "2.0.0",
    "url": "https://github.com/cure53/DOMPurify"
  },
  {
    "name": "svg.js",
    "version": "2.7.1",
    "url": "https://github.com/svgdotjs/svg.js"
  }
]
```

**Comment by KZzizzle** [comment]
Run button was hidden and workbench panels were not expandable - seems to be a problem mainly for small screens.

**Comment by KZzizzle** [comment]
Changes in the collapsing frames has closed this issue, from TB Meeting 02.03.2021
